src/classify/classifier/svm_wrapper.cpp

```diff
@@ -90,62 +171,30 @@ confusion_matrix svm_wrapper::test(const std::vector<doc_id>& docs)
 
     // extract answer
     confusion_matrix matrix;
-    std::ifstream in("svm-predicted");
-    std::string str_val;
-    for (auto& d_id : docs)
     {
-        // we can assume that the number of lines in the file is equal to the
-        // number of testing documents
-        std::getline(in, str_val);
-        uint32_t value = std::stoul(str_val);
-        class_label predicted = idx_->class_label_from_id(label_id{value});
-        class_label actual = idx_->label(d_id);
-        matrix.add(predicted, actual);
+        std::ifstream in("svm-predicted");
```

Comment:
Fixed this (and several others of this; I think this code might be one of the older parts of MetA!) in a3a1f5c.